GenerateProjects.bat

```diff
@@ -1,2 +1,5 @@
+@echo off
 call vendor\bin\premake\premake5.exe vs2017
-PAUSE
\ No newline at end of file
+IF %ERRORLEVEL% NEQ 0 (
```

Comment:
Why is this PR doing multiple things that supposed to? This code is from another PR.

Reply:
This is not done by this PR. This was done in TheCherno/Hazel#83. We just upstreamed Hazel to be up to date with the latest changes.